### ISSUE TITLE
Create Ansible playbook/role for PHP Censor

### ISSUE THREAD

**Issue by corpsee** [php-censor/php-censor#129]
Create Ansible playbook/role for PHP Censor.

**Comment by ketchoop** [comment]
@corpsee it's really good point. But it would be easily to do, if PHPCensor have rpm/deb packages.

**Comment by corpsee** [comment]
@ketchoop Good idea :) I will try to create playbook/role and see what I can do. May be it will system package at the end.
